src/jsdoc_transformer.ts

```diff
@@ -431,14 +431,24 @@ export function removeTypeAssertions(): ts.TransformerFactory<ts.SourceFile> {
   };
 }
 
+/**
+ * @return true if node lexically contains an 'async' function.
```

Comment:
Google style is to prefer text description over the return tag. Also, call out that it's the entire subtree?

`containsAsync returns true if node or any of its descendants lexically contain ...`

Reply:
I think repeating the function name is Go style, right?  In any case, fixed.

Reply:
The function name indeed is Go style, I guess in JS you'd do "Returns ..." (but I think the Go style is kosher as well, and nicer).